src/lambda/handler-runner/docker-runner/DockerContainer.js

```diff
@@ -13,30 +13,34 @@ export default class DockerContainer {
   static #dockerPort = new DockerPort()
 
   #containerId = null
+  #containerPort = null
   #env = null
   #functionKey = null
   #handler = null
   #imageNameTag = null
   #image = null
   #port = null
+  #host = null
 
-  constructor(env, functionKey, handler, runtime) {
+  constructor(env, functionKey, handler, runtime, host) {
     this.#env = env
     this.#functionKey = functionKey
     this.#handler = handler
     this.#imageNameTag = this._baseImage(runtime)
     this.#image = new DockerImage(this.#imageNameTag)
+    this.#host = host
   }
 
   _baseImage(runtime) {
     return `lambci/lambda:${runtime}`
   }
 
   async start(codeDir) {
-    const [, port] = await Promise.all([
-      this.#image.pull(),
-      DockerContainer.#dockerPort.get(),
-    ])
+    // const [, port] = await Promise.all([
+    //   this.#image.pull(),
+    //   DockerContainer.#dockerPort.get(),
+    // ])
```

Comment:
I don't think `this.#image.pull()` should be commented out.
```suggestion
    await this.#image.pull()
```